lib/rules/order-in-components.js

```diff
@@ -1,60 +1,72 @@
 'use strict';
 
-var utils = require('../utils/utils');
 var ember = require('../utils/ember');
+var reportUnorderedProperties = require('../utils/property-order').reportUnorderedProperties;
+
+const ORDER = [
+  'service',
+  'property',
+  'single-line-function',
+  'multi-line-function',
+  'observer',
+  'lifecycle-hook',
+  'actions',
+  'method',
+  'unknown',
+];
+
+const NAMES = {
+  'service': 'service injection',
```

Comment:
What do you think about moving all those names to a separate file and using them inside `property-order.js`? It looks like there are some unnecessary duplications in current solution.